README.md

```diff
@@ -1,2 +1,78 @@
-# yoti-node-sdk-sandbox
-Yoti Node SDK Sandbox
+# Yoti NodeJS Sandbox SDK
+
+Welcome to the Yoti NodeJS Sandbox. This repo contains the tools you need to test your NodeJS back-end integration.
+
+## Requirements
+
+### Node version
+Please refer to [Travis](https://travis-ci.com/getyoti/yoti-node-sdk-sandbox) to see all compatible Node versions.
+
+## Installing the Sandbox
+
+To import the Yoti SDK inside your project, you can use your favourite dependency management system.
+If you are using NPM, you can use the following command to set the Yoti SDK as a dependency:
+
+```shell
+npm install @getyoti/sdk-sandbox
+```
+
+## Configuration
+
+* `CLIENT_SDK_ID` is the SDK identifier generated by Yoti Hub in the Key tab when you create your app. Note this is not your Application Identifier which is needed by your client-side code.
+
+* `path/to/your-application-pem-file.pem` is the path to the application pem file. It can be downloaded only once from the Keys tab in your Yoti Hub.
+
+Please do not open the pem file as this might corrupt the key and you will need to create a new application.
+
+## Profile Token Creation
+
+```javascript
+const fs = require('fs');
+
+const {
+  SandboxProfileClientBuilder,
+  SandboxAgeVerificationBuilder,
+  TokenRequestBuilder,
+} = require('@getyoti/sdk-sandbox');
+
+const CLIENT_SDK_ID = 'your sdk id';
+const PEM = fs.readFileSync(__dirname + '/keys/your-application-pem-file.pem');
+
+const sandboxProfileClient = new SandboxProfileClientBuilder()
+  .withClientSdkId(CLIENT_SDK_ID)
+  .withPemString(PEM)
+  .build();
+
+const ageVerification = new SandboxAgeVerificationBuilder()
+  .withDateOfBirthString('1980-01-01')
+  .withAgeOver(18)
+  .build();
+
+const tokenRequest = new TokenRequestBuilder()
+  .withRememberMeId('some remember me ID')
+  .withGivenNames('some given names')
+  .withFamilyName('some family name')
+  .withFullName('some full name')
+  .withDateOfBirthString('1980-01-01')
+  .withAgeVerification(ageVerification) {
+  .withGender('some gender')
+  .withPhoneNumber('some phone number')
+  .withNationality('some nationality')
+  .withStructuredPostalAddress(JSON.stringify({
+    building_number: 1,
+    address_line1: 'some address',
+  }))
+  .withBase64Selfie('some base64 encoded selfie')
+  .withEmailAddress('some@email')
+  .withDocumentDetails('PASSPORT USA 1234abc')
+  .build();
+
+sandboxProfileClient.setupSharingProfile(tokenRequest)
+  .then((response) => {
+    const token = response.getToken();
+    // Use token to get activity details.
```

Comment:
Could we explicitly add the code here to get the activity details too for ease of use? 
I've added the following for .NET:
```cs
var yotiClient = new YotiClient(clientSdkid, stream);
ActivityDetails activityDetails = yotiClient.GetActivityDetails(sandboxOneTimeUseToken);

Assert.AreEqual("some@email", activityDetails.Profile.EmailAddress.GetValue());
```

Reply:
Are we making an assumption that the token will be used inline like this, as opposed to being passed to an application as a URL parameter?

I'll update as this gives better context 👍

Reply:
Added in b038ef3